### DISCUSSION TITLE
difficulty names getting replaced where they shouldn't

### DISCUSSION THREAD

**Discussion by camellirite** [ppy/osu#16887] [Q&A] [answered]
![osu!_wuJjVoPjSZ](https://user-images.githubusercontent.com/23125409/154210358-8f14546e-ea89-4fba-a560-a84f1802f267.png)

it should be "Neru - Lost One no Goukoku (eveless) mapped by goodbye"

for context, goodbye's username used to be eveless

i found this in the playlist "osu! Beatmap Spotlights (Winter 2022): Playlist A"

**Comment by bdach** [comment, marked as answer]
that's not the difficulty name. difficulty name is in square brackets and got truncated due to not fitting in this case.

if there's an issue here then it's that the username is printed twice on the playlist item unnecessarily.

**Reply by peppy** [reply]
Tracking via https://github.com/ppy/osu/issues/16888.

**Comment by camellirite** [comment]
that's true, I don't see a reason why it should be there twice
